README.md

```diff
@@ -1,16 +1,32 @@
-# Title
+# Bookend: Git Workflow
 
-## Learning Goals
+## Looking Back
 
--SWBAT 1
--SWBAT 2
+Over the past several lessons, we built on the foundation of our Git basics
+knowledge to integrate Git more effectively in our workflow. We learned:
 
-## Introduction
+- The primary role of branching
+- How to create a branch
+- How to add and commit changes
+- How to push branches
+- Identify local and remote branches
+- How to pull branches
+- How to merge branches
+- How to delete branches
+- How to review a repository’s history with `git log`
+- How to focus on certain changes with `git diff`
+- How to unstage changes with `git reset`
+- How to revert changes with `git checkout`
 
-## SWBAT 1
+As was true when we first learned Git, don't worry if all of these concepts or
+actions don't yet feel like second nature for you or you have to check back with
+your resource guides. The more you work with them, and the more you see how they
```

Comment:
I would hyperlink "resource guides" to a good git cheat sheet (we probably have one in a lesson somewhere).